### PR TITLE
Technical updates to LIP-1

### PR DESCRIPTION
**Pull request by gdanezis**
* Updated example structures with latest formats and constraints.
* Clarified that the "_writes" field is also a ref_ID -> cid map.

### CODE REVIEW

File: lips/lip-1.mdx [hunk 1]
```diff
@@ -191,17 +201,20 @@ All responses to a CommandRequestObject are in the form of a CommandResponseObje
 |-------	     |------	|-----------	|-------------	|
 | _ObjectType    | str      | Y             | The fixed string `CommandResponseObject`. |
 | status         | str      | Y             | Either `success` or `failure`. |
-| error          | List of [OffChainErrorObject](#offchainerrorobject) | N | Details on errors when status == "failure"
+| error          | List of [OffChainErrorObject](#offchainerrorobject) | N | Details on errors when status == "failure" |
+| cid            | str      | N | The command identifier to which this is a response. |
```
Comment (kphfb) [inline lips/lip-1.mdx:205]: Why is this not required?

Comment (kphfb) [reply lips/lip-1.mdx:205]: Ah nvm

File: lips/lip-1.mdx [hunk 2]
```diff
@@ -302,16 +319,17 @@ For a travel rule data exchange, the [command_type](basic_building_blocks.md#com
...
-
-| _writes | list of str | Y | For a payment Command, the `_writes` field may only be a single value since a payment Command only operates upon one Object.  This field specifies the `reference_id` of the Object bring written. |
+| _writes | JSON object map | Y | For a payment command, the `_writes` field may only be a single key-value pair since a payment command only operates upon one object.  This field maps the `reference_id` of the object being written to its new version. |
```
Comment (kphfb) [inline lips/lip-1.mdx:322]: Legal wanted Command and Version and Object to be capitalized since they are defined values in the definitions list

**Review by dahliamalkhi — APPROVED** [review]
seems mostly cosmetic/legal stuff.

**Pull request merged**